action.yml

```
@@ -14,6 +14,12 @@ inputs:
   workspaces:
     description: "Paths to multiple Cargo workspaces and their target directories, separated by newlines"
     required: false
+  maxRetryAttempts:
```

```suggestion
  max-retries:
```

Maybe? To rather use dashed names to be consistent wtih the other options.